utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/skeletons/SkeletonExampleCodeReact.jsx

```diff
@@ -0,0 +1,25 @@
+import { ExampleCodeReactProp } from '../../../../../sandbox/ExampleCodeReactProp';
+
+/** @typedef {import('utah-design-system-website').SkeletonExamplePropsShape} SkeletonExamplePropsShape */
+
+/**
+ * @param {object} props
+ * @param {{props: SkeletonExamplePropsShape}} props.state
+ * @returns {import('react').JSX.Element}
+ */
+export function SkeletonExampleCodeReact({
+  state: {
+    props: {
+      className,
+      style,
+    },
+  },
+}) {
+  return (
+    <>
+      &lt;Skeleton
+      <ExampleCodeReactProp displayProp={className ? `className="${className}"` : null} indentLevel={1} />
```

Comment:
This renders: 
```
<Skeleton  className="awefawfe"
  style="skeleton--block" />
```
Spinner adds a blank line after the open tag so className is on its own line:
```
      &lt;Spinner
      {
        (className || id || size || strokeWidth || value)
          ? <br />
          : null
      }
      <ExampleCodeReactProp displayProp={className ? `className="${className}"` : null} indentLevel={1} />
```